### PR TITLE
change hanadb_exporter port to 9668

### PR DESCRIPTION
**Pull request by MalloZup**
see https://github.com/SUSE/hanadb_exporter/pull/55 
https://github.com/SUSE/hanadb_exporter/pull/56/files

### CODE REVIEW

**Comment by MalloZup** [comment]
thx! @diegoakechi can you merge it? thx

**Comment by MalloZup** [comment]
https://github.com/SUSE/hanadb_exporter/pull/56/files

File: pillar_examples/automatic/hana/hana.sls [hunk 1]
```diff
@@ -31,7 +31,7 @@ hana:
           database: SYSTEMDB
     {% if grains.get('monitoring_enabled', False) %}
       exporter:
-        exposition_port: 8001
+        exposition_port: 9668
```
Comment (arbulu89) [inline pillar_examples/automatic/hana/hana.sls]: Change the port we use for the QAS HANA in this file.
Now it's defined to use `8002`, I would put `9669` to make it lineal

**Comment by arbulu89** [comment]
@MalloZup 
Don't merge this PR until https://github.com/SUSE/saphanabootstrap-formula/pull/68 is merged.

Finally, when `saphanabootstrap-formula` reaches our OBS factory repo and `hanadb_exporter` is in `server:monitoring` we can merge finally this branch to `master`, and them bring the changes to `develop`

File: pillar_examples/automatic/hana/hana.sls (hunk 1 above)
Comment (stefanotorresi) [reply pillar_examples/automatic/hana/hana.sls]: what's QAS HANA?

**Comment by stefanotorresi** [comment]
@arbulu89 this still needs rebasing against master, anyway

File: pillar_examples/automatic/hana/hana.sls (hunk 1 above)
Comment (arbulu89) [reply pillar_examples/automatic/hana/hana.sls]: The 3rd HANA instance used of Quality Assurance in this case.

This 3rd HANA creates a cost-optimized scenario, where together with the 1st Cluster (primary/secondary 2 HANA Production instances) we have other 3rd HANA instance in the 2n Cluster Node (So we have 2 HANA instances in the 2nd node).

In the cost-optimized scenario the Secondary HANA (the 2nd) is running in a limited resources scope, and in the case of a failover to the 2nd node, the QAS instance (3rd HANA) will be totally stopped to give room the the Production HANA.

**Comment by arbulu89** [comment]
> @arbulu89 this still needs rebasing against master, anyway

The PR is against Master. And I guess that that's normal as the hanadb_exporter (now prometheus-hanadb_exporter) has breaking changes)

**Pull request merged**